Storage/api.h

```diff
@@ -219,6 +219,13 @@ STORAGE_HANDLERS_MAP_ENTRY GenerateRESTfulHandler(STORAGE& storage, const std::s
 }
 
 }  // namespace impl
+template <bool CALL>
```

Comment:
It could be placed in `Bricks/template/`. Though I still don't get the idea of reversed logic :)